spec/compiler/codegen/arithmetics_spec.cr

```diff
@@ -2,9 +2,17 @@ require "../../spec_helper"
 
 {% if flag?(:bits64) %}
 SupportedInts = [UInt8, UInt16, UInt32, UInt64, UInt128, Int8, Int16, Int32, Int64, Int128]
```

Comment:
Shouldn't these constants be capitalized and in snake_case?

Reply:
no, it's not mandatory. It is valid and used in other places, like `c_struct_spec.cr` to mention one.